dqc_us_rules/dqc_us_0015.py

```diff
@@ -72,7 +73,7 @@ def filter_negative_number_facts(val, blacklist_concepts):
     # other filters before running negative numbers check
     # numeric_facts has already checked if fact.value can be made into a number
     facts_to_check = [
-        f for f in numeric_facts if float(f.value) < 0 and
+        f for f in numeric_facts if f.xValid >= VALID and f.xValue < 0 and
```

Comment:
Still confused why you aren't doing:

    if f.xValid == VALID

I don't understand the `>` part of this.


That being said in python you can drop the `is not None` on null checks as long as they aren't integers. So this could be reduced to:

    f for f in numeric_facts if f.xValid >= VALID and f.xValue < 0 and
    f.concept.type and f.concept and f.isNumeric